tools/test.py

```diff
@@ -87,6 +87,10 @@ def main():
     if cfg.get('cudnn_benchmark', False):
         torch.backends.cudnn.benchmark = True
     cfg.model.pretrained = None
+    if cfg.model.get('neck', False):
```

Comment:
`cfg.model.get('neck')` will return None if neck does not exist, thus we can omit the default value.